### DISCUSSION TITLE
Trying to build 1.8.0 from GitHub in Visual Studio; missing closedir() function.

### DISCUSSION THREAD

**Discussion by ViridianGames** [exult/exult#314] [General]
I pulled 1.8.0 from GitHub in preparation for a pull request but I can't build the latest on Windows in Visual Studio because there's a function on line 442 of Utils.cc called closedir() which I can't seem to resolve in Windows.  Including dirent.h does not allow the function to be used like I would have expected it.

Any help would be appreciated.

**Comment by ViridianGames** [comment]
Ugh, as SOON as I post I see the issue.  I wasn't on 1.8.x branch, I was on master.  That's what I get for trying to do this without enough caffeine.

**Comment by wench** [comment]
I have a fix for this locally for master will push it shortly

**Reply by ViridianGames** [reply]
Oh, thank you!

**Reply by wench** [reply]
Done in d640e3abb2aba37a3cf33e748ca5dae54ee7347b

**Reply by ViridianGames** [reply]
Again, thank you.
